phpseclib/System/SSH/Agent.php

```diff
@@ -128,7 +128,7 @@ function __construct()
                 break;
             default:
                 user_error('SSH_AUTH_SOCK not found');
-                return false;
+                return;
```

Comment:
Yes, http://stackoverflow.com/a/6849691/1879087 + as I recall this can be a problem in PHP 7 (plugins community hints)